src/index.ts

```diff
@@ -1,4 +1,17 @@
 import App from './App';
+import debug from 'debug';
+
+const error = debug('team-link:error');
+
+process.on('unhandledRejection', () => {
+    error('Unhandled promise rejection');
+    throw new Error('Unhandled promise rejection');
```

Comment:
If we log the error in our own way, do we really need to throw an error here? Ah, and if we do it this way, we can completely lose information about the original error.